docs/src/pages/[platform]/theming/style-props/examples/ShadowExamples.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button, useTheme } from '@aws-amplify/ui-react';
+
+//use any CSS value
+export const ShadowStylePropExample = () => {
+  return (
+    <Button boxShadow="rgba(13, 26, 38, 0.25) 0px 4px 12px 0px">
```

Comment:
I can't see the `boxShadow` on dark mode. This may not be an issue we can solve in this PR though

Reply:
Could we put a background behind the example so that you can always see it?

Reply:
Oh wait, I think the box shadow design tokens automatically flip in dark mode. Can you use one of those instead?

Reply:
I've switched the example showing to use the design tokens, however the same issue exists.  I believe this is because the shadow colors don't have a color override in the dark mode theme. Perhaps we can add shadow color overrides to the list of fast followup items.